Formula/groonga.rb

```diff
@@ -41,6 +43,11 @@ class Groonga < Formula
     sha256 "bc83d1e5e0f32d4b95e219cb940a7e3f61f0f743abd3bd47c2d436a34e503870"
   end
 
+  resource "stemmer" do
+    url "https://github.com/snowballstem/snowball.git",
+      :revision => "71936098048f915a797de990776564c924d40b6a"
```

Comment:
I wish. The official notes from the new Snowball page suggest that if a "stable" source distribution is required, one should use the old download available at http://snowball.tartarus.org/download.html, which is from 2014 and nonetheless still contains no numeric version number. There has been no update on @snowballstem/snowball#7 for over a year. I could ping them, however.

It appears that the [dovecot](https://github.com/Homebrew/homebrew-core/blob/master/Formula/dovecot.rb#L27) and [sphinx](https://github.com/Homebrew/homebrew-core/blob/master/Formula/sphinx.rb#L28) formulae have run into this same problem and also chose to pick an arbitrary commit for the resource.